### PR TITLE
GitHub workflow configuration to deploy static assets

### PR DESCRIPTION
**Pull request by skambalin**
> _The goal of the pull request is not only to propose a working code as a solution to the requirement.  
> It is a tool for you to give context and explanation of your approach to the problem,  
> and transfer the knowledge to your team members._

## Checklist (Mandatory):

- [x] I have a ticket for this PR ([link](https://www.notion.so/cere/Add-WDIO-test-runs-configuration-to-Game-SDK-CI-d82bc3b81a014fda8c39c1fc3b0e33e9?pvs=4))
- [ ] I have added a record to the `CHANGELOG.md`
- [ ] I have tested new code locally
- [ ] I have done a smoke test of the larger feature locally
- [x] I have merged the `dev` (parent) branch into my feature branch and resolved all conflicts
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works

## Checklist (Optional):

- [x] I have written the code according to the adopted [style guide](https://github.com/cere-io/fe-guidelines)?

## Additional notes

- Add env files and workflows configuration for each env

### CODE REVIEW

**Review by evgeny-s — APPROVED** [review]

File: .github/workflows/dev.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+jobs:
+  build_and_deploy:
+    uses: cere-io/reusable-workflows/.github/workflows/deploy-to-cloudfront.yaml@master
+    with:
+      build_container: 'node:16-buster-slim'
+      install_packages_command: 'cp .env.dev .env; npm ci'
+      build_command: 'npm run build'
+      path_to_static_files_to_upload: 'build'
+      aws_account_id: ${{ vars.DEV_CORE_AWS_ACCOUNT_ID }}
+      s3_bucket_name: 'games-sdk.core-dev.aws.cere.io'
```
Comment (krolol) [inline .github/workflows/dev.yaml:21]: games-sdk-dev.core-dev.aws.cere.io

File: .github/workflows/stage.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+jobs:
+  build_and_deploy:
+    uses: cere-io/reusable-workflows/.github/workflows/deploy-to-cloudfront.yaml@master
+    with:
+      build_container: 'node:16-buster-slim'
+      install_packages_command: 'cp .env.stage .env; npm ci'
+      build_command: 'npm run build'
+      path_to_static_files_to_upload: 'build'
+      aws_account_id: ${{ vars.STG_CORE_AWS_ACCOUNT_ID }}
+      s3_bucket_name: 'games-sdk.core-stage.aws.cere.io'
```
Comment (krolol) [inline .github/workflows/stage.yaml:21]: games-sdk-stg.core-stage.aws.cere.io

**Review by krolol — CHANGES_REQUESTED** [review]

File: .github/workflows/dev.yaml (hunk 1 above)
Comment (skambalin) [reply .github/workflows/dev.yaml]: Updated

File: .github/workflows/stage.yaml (hunk 2 above)
Comment (skambalin) [reply .github/workflows/stage.yaml]: Updated

**Pull request merged**